Source/WebCore/platform/graphics/mac/controls/ButtonMac.mm

```diff
@@ -49,25 +49,24 @@
 {
     // Buttons really only constrain height. They respect width.
     static const IntSize cellSizes[] = {
-        { 0, 21 },
-        { 0, 18 },
-        { 0, 15 },
+        { 0, 20 },
+        { 0, 16 },
+        { 0, 13 },
         { 0, 28 }
     };
```

Comment:
maybe add a comment to keep this in sync with ThemeMac.mm (although it would be nice it wasn't duplicated at all)

Reply:
@shallawa Do you have a bug tracking this work: https://github.com/WebKit/WebKit/pull/8062#discussion_r1062743870?